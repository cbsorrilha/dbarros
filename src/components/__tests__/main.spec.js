import React from "react"
import renderer from "react-test-renderer"
import Main from '../main';


describe('Snapshot testing the Main', () => {
    it("renders correctly", () => {
        const props = {}
        const tree = renderer.create(<Main {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
