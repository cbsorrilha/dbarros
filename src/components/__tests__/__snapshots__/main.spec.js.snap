// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the Main renders correctly 1`] = `
<main
  className="sc-AykKC kpdnlP"
/>
`;
